src/index.ts

```diff
@@ -110,10 +110,9 @@ if (!argv.file) {
 const fakeDefinitionAST = readAST(path.join(__dirname, 'fake_definition.graphql'));
 const corsOptions = {}
 
-if (argv.co) {
-  corsOptions['origin'] =  argv.co
-  corsOptions['credentials'] =  true
-}
+// Currently no way to turn CORS off entirely, but that is probably fine
```

Comment:
If someone reports an issue we can always add `--no-cors`.
Can you please remove this comment.